renderers/games.py

```diff
@@ -61,23 +58,22 @@ def render(self):
     is required.
     """
     self.canvas.Fill(*ledcolors.scoreboard.fill)
-    current_game_index = self.__get_game_from_args()
-    game = self.games[current_game_index]
+    # current_game_index = self.data.game_index_for_preferred_team()

```

Comment:
remove commented lines

Reply:
I went ahead and removed this comment but, ideally we don't need this file at all. It's been replaced with the MainRenderer. Was just keeping it around until we were sure everything worked.